### PR TITLE
Javascript/javascript3/week3

### PR DESCRIPTION
**Pull request by dinythomas89**

### CODE REVIEW

File: javascript/javascript3/week3/shopping-cart.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        price.innerHTML = "Total price: " + total;
+    }
+    getUser() {
+        const fetchPromise = fetch('https://jsonplaceholder.typicode.com/users/1')
+            .then(response => response.json())
+            .then(data => {
+                return data;
+            });
+        return fetchPromise
+    }
```
Comment (Swapna-Sahu) [inline javascript/javascript3/week3/shopping-cart.js:75]: getUser is from another task . It will e good if u keep it in different file.

**Review by Swapna-Sahu — CHANGES_REQUESTED** [review]
Circle and promise is working . But shopping cart is not according to task. Please have a look in the task and use all files .

File: javascript/javascript3/week3/shopping-cart.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        });
+        return sum;
+    }
+    renderProducts() {
+        this.getUser().then((data) => userName.innerHTML = data.name);
+        this.products.forEach(element => {
+            const li = document.createElement('li');
+            ul.appendChild(li);
+            const h2 = document.createElement('h3');
+            h2.innerHTML = element.name;
```
Comment (Swapna-Sahu) [inline javascript/javascript3/week3/shopping-cart.js]: you are creating h3 but your variable name id h2. It should be same for better naming convention.

**Review by Swapna-Sahu — APPROVED** [review]
Few comments for better coding but looks good to me

File: javascript/javascript3/week3/shopping-cart.js (hunk 2 above)
Comment (dinythomas89) [reply javascript/javascript3/week3/shopping-cart.js]: Thanks Swapna for the feedback.

**Pull request merged**